README.md

```diff
@@ -8,7 +23,8 @@ Note - if you are pulling, look at the [branches section](#branches)
 Firefly builds are available on Docker and the notes for using it are on the [Docker Page](https://hub.docker.com/r/ipac/firefly).
 
 
-### Firefly Framework
+## Firefly Framework
+
 Firefly is being reused for various archives because it is a framework of archive components. This has allowed us now to implement multiple archive systems at a fraction of the cost of the first system, Spitzer Heritage Archive. Firefly has undergone an extensive re-write since then, adopted React/Redux framework for the client side, making it much  easier to work with other JavaScript libraries. Recently we have adopted Plotly.js graphing library.
```

Comment:
Remove "now".
Change "since then, adopted" to "since its initial release, notably adopting the"

Move the "Plotly" sentence to the "Scientific Plotting" section below, as "Firefly's general scientific visualizations are based on the Plotly.js graphing library."